test/server_test.lua

```diff
@@ -19,6 +19,14 @@ local server = Server:new({
     net_box_port = 3133,
 })
 
+local SOCKET_DIR = require('fio').cwd()
+local server_with_socket = Server:new({
```

Comment:
I don't think that there is a need in another `Server` when old `server` can be restarted with new arguments.

Reply:
Env variables are built during initialize function. So, `TARANTOOL_LISTEN` could not be passed to the instance file and a socket could not be created. But I can create a new server in the test, I think it would be more wonderful. What is your opinion, @Steap2448 ?